sdk/nodejs/apps/v1/ControllerRevision.ts

```diff
@@ -106,6 +106,14 @@ import { getVersion } from "../../version";
               opts.version = getVersion();
           }
 
-          super(ControllerRevision.__pulumiType, name, props, opts);
+          const _opts = pulumi.mergeOptions(opts, {
```

Comment:
Not a big deal - but since we're already modifying `opts` - may as well just have this be `opts = ...`.